.github/labeler.yml

```diff
@@ -0,0 +1,3 @@
+compute:  
```

Comment:
Azure repo has "Compute" label.  Can we change it Compute to match the existing label?

Reply:
label updated to computeReview